### ISSUE TITLE
[Bug]: Update for sensor.xxxx fails 

### ISSUE THREAD

**Issue by andrewjswan** [custom-components/ble_monitor#1310]
### What happened?

A lot of RSSI error in log

### Relevant log output

```shell
2024-02-08 23:45:53.498 ERROR (MainThread) [homeassistant.helpers.entity] Update for sensor.ble_battery_kitchen_clock fails
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 942, in async_update_ha_state
    await self.async_device_update()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1259, in async_device_update
    await self.async_update()
  File "/config/custom_components/ble_monitor/sensor.py", line 817, in async_update
    self._extra_state_attributes["rssi"] = round(sts.mean(self.rssi_values))
                                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/statistics.py", line 486, in mean
    raise StatisticsError('mean requires at least one data point')
statistics.StatisticsError: mean requires at least one data point
2024-02-08 23:45:53.501 ERROR (MainThread) [homeassistant.helpers.entity] Update for sensor.ble_battery_alarm_clock fails
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 942, in async_update_ha_state
    await self.async_device_update()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1259, in async_device_update
    await self.async_update()
  File "/config/custom_components/ble_monitor/sensor.py", line 817, in async_update
    self._extra_state_attributes["rssi"] = round(sts.mean(self.rssi_values))
                                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/statistics.py", line 486, in mean
    raise StatisticsError('mean requires at least one data point')
```

**Comment by Ernst79** [comment]
Did this start after the update to HA 2024.2?

**Comment by andrewjswan** [comment]
I can’t say for sure, I didn’t notice it before, but I noticed it on 2024.2.0

**Comment by Ernst79** [comment]
Fixed in 12.9.2
